modules/tinymce/src/plugins/image/test/ts/browser/DialogUpdateTest.ts

```diff
@@ -27,6 +38,28 @@ UnitTest.asynctest('browser.tinymce.plugins.image.DialogUpdateTest', (success, f
         }),
         cSubmitDialog(),
         cAssertCleanHtml('Checking output', '<p><img src="#2" /></p>')
+      ]),
+
+      Log.chainsAsStep('TINY-6611', 'Setting src to empty should remove the existing dimensions settings', [
+        Chain.inject(editor),
+        ApiChains.cSetContent('<p><img src="https://www.google.com/logos/google.jpg"  width="200" height="200"/></p>'),
+        ApiChains.cSetSelection([ 0 ], 0, [ 0 ], 1),
+        ApiChains.cExecCommand('mceImage', true),
+        cWaitForDialog(),
+        Chain.fromParent(Chain.identity, [
+          cAssertInputValue(generalTabSelectors.src, 'https://www.google.com/logos/google.jpg'),
+          cAssertInputValue(generalTabSelectors.height, '200'),
+          cAssertInputValue(generalTabSelectors.width, '200')
+        ]),
+        cSetInputValue(generalTabSelectors.src, ''),
+        cFakeEvent('change'),
+        Chain.fromParent(Chain.identity, [
+          cAssertInputValue(generalTabSelectors.height, ''),
+          cAssertInputValue(generalTabSelectors.width, '')
+        ]),
+        cSubmitDialog(),
+        Chain.inject(editor),
```

Comment:
Instead of injecting the editor again, I believe you could wrap the set input value in an isolated chain. eg:
```
Chain.fromIsolatedChains([
  cSetInputValue(generalTabSelectors.src, ''),
  cFakeEvent('change'),
]),

Reply:
Perfect. Done.